lxd/instance/drivers/driver_lxc.go

```diff
@@ -2564,6 +2564,27 @@ func (d *lxc) onStart(_ map[string]string) error {
 	if d.expandedConfig["limits.network.priority"] != "" {
 		go func(d *lxc) {
 			d.fromHook = false
+
+			timeout := 10 * time.Second
```

Comment:
Maybe it makes sense to use the same timeout pattern across the codebase. Tom once posted a good suggestion to safe resources here https://github.com/canonical/lxd/pull/11993#discussion_r1262190102. The final code is here https://github.com/canonical/lxd/blob/main/lxd/instance/drivers/driver_qemu.go#L7529

Reply:
Absolutely agree. Will do.